language/move-lang/src/cfgir/liveness/state.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//**************************************************************************************************
+// Abstract state
+//**************************************************************************************************
+
+use crate::{cfgir::absint::*, parser::ast::Var};
```

Comment:
Probably worth defining a `SetDomain<T>` utility at some point and just using `SetDomain<Var>` here at some point, but by no means urgent.